client.go

```diff
@@ -22,6 +23,10 @@ type Client struct {
 	debugLogFunc LogFunc
 }
 
+func LintError() {
+	fmt.Printf("I'm going to error: %d", "no bueno")
```

Comment:
**[govet]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
arg "no bueno" for printf verb %d of wrong type: string